package.json

```diff
@@ -9,6 +9,9 @@
   "scripts": {
     "test": "truffle test",
     "coverage": "./node_modules/.bin/solidity-coverage",
+    "networks-extract": "CONF_FILE='../../../../../src/manage_network_info/conf.js' node node_modules/@gnosis.pm/util-contracts/src/extract_network_info.js",
```

Comment:
Is there no better way to do this? E.g. somehow knowing what the package.json folder is and going from there?

Reply:
I was searching for a good solution but I could not find it. But with this great tip, I got it! ;)